HowTos/create_vpc.rst

```diff
@@ -1,39 +1,58 @@
 .. meta::
    :description: VPC Network CIDR Management Tool
-   :keywords: Aviatrix VPC Tracker, AWS VPC
+   :keywords: Aviatrix VPC Tracker, AWS VPC, AZURE VNet
 
 ###################################
 Create a VPC
 ###################################
 
-Use this tool to create a `VPC <https://www.aviatrix.com/learning/glossary/vpc.php>`_ that consists of a public subnet and a private subnet in two AZs and an IGW in the region and account of your choice. 
+Use this tool to create a `VPC <https://www.aviatrix.com/learning/glossary/vpc.php>`_ in AWS or a `VNet <https://a.aviatrix.com/learning/glossary/vnet.php>`_ in AZURE in the region and account of your choice. In addition, starting from 6.1, this tool creates multiple route tables associated with public and private subnets. One use case is to allow traffic load balancing when Aviatrix Spoke gateways are deployed. 
```

Comment:
By multiple RT, I see that each subnet created points to its own distinct RT. 
Multiple RT is not shared; each one has its own RT.